be1-go/cli/cli.go

```diff
@@ -69,10 +71,12 @@ func (s *ServerConfig) newHub(l *zerolog.Logger) (hub.Hub, error) {
 		s.ServerAddress = fmt.Sprintf("ws://%s:%d/server", s.PublicAddress, s.ServerPort)
 	}
 
-	path := "./database-a/" + sqlite.DefaultPath
-
-	if s.ClientPort == 9002 {
-		path = "./database-b/" + sqlite.DefaultPath
+	if s.DatabasePath == "" {
+		if s.ClientPort == 9002 {
+			s.DatabasePath = "./database-b/" + sqlite.DefaultPath
+		} else {
+			s.DatabasePath = "./database-a/" + sqlite.DefaultPath
+		}
 	}
```

Comment:
Could you add the datapath to both configFile1 and configFile2 and then remove the corresponding lines?

Reply:
Done